logs/command.go

```diff
@@ -46,13 +49,13 @@ func NewLogsCommand() *cli.Command {
 		Usage: "command-line search for SolarWinds Observability log management service",
 		Flags: flags,
 		ArgsUsage: `
-	
+
 EXAMPLES:
    swo logs something
    swo logs 1.2.3 Failure
    swo logs -s ns1 "connection refused"
-   swo logs "(www OR db) (nginx OR pgsql) -accepted"
-   swo logs -g <SWO_GROUP_ID> "(nginx OR pgsql) -accepted"
+   swo logs -f "(www OR db) (nginx OR pgsql) -accepted"
+   swo logs -f -g <SWO_GROUP_ID> "(nginx OR pgsql) -accepted"
```

Comment:
SWO_GROUP_NAME

Reply:
Can you fix GROUP_ID references in general?

Reply:
done